src/ruby/yast/i18n.rb

```diff
@@ -1,3 +1,4 @@
+# coding: utf-8
```

Comment:
NP: IIRC the UTF-8 encoding is the default in Ruby 2.0+, this line is not need. Or did I overlok something?

Reply:
no, really no longer needed for newer ruby

Reply:
My Emacs mode let this slip through, I'll fix that